### ISSUE TITLE
Incorrect name is provided for close button as image: A11y_VS Code extension for API Center_Open API Documentation_Close_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#111]
### GitHub Tags
#A11yTCS; #A11ySev2; #Mac; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Name Role Value; #AILimited; #A11yMAS; #device;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Demo conference API screen will appear.
8. Tab till authorize button and press enter
9. Verify whether the correct name is provided for close button as image or not.

### Actual Result:
Incorrect name is provided for close button as image.

### Expected Result:
Correct name as a button should be provided for close button.

### User Impact:
Screen reader users will get the confuse if the incorrect name is provided.

### Attachment:

**Comment by AnKushSingh05** [comment]
@formulahendry Can you please provide us the standalone sample app or Code pen link to route this bug to external team.

**Comment by AnKushSingh05** [comment]
IRT bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222927

**Comment by Yash14j** [comment]
### Github tags:
#External;#External:SwaggerUI;#eDAD:3p:222927;ThirdParty;

**Comment by formulahendry** [comment]
> [@formulahendry](https://github.com/formulahendry) Can you please provide us the standalone sample app or Code pen link to route this bug to external team.

@AnKushSingh05 Here is the Code pen link: https://codepen.io/formulahendry/pen/xxvQGaQ

**Comment by AnKushSingh05** [comment]
External bug: https://github.com/swagger-api/swagger-ui/issues/10320

**Comment by AnKushSingh05** [comment]
Adding the tag #ResolvedByTT;
